frontend/app/openproject-app.js

```diff
@@ -44,7 +44,6 @@ require('angular-animate');
 require('angular-aria');
 require('angular-modal');
 
-// require('angular-i18n/angular-locale_en-us');
 if (I18n.locale === 'de') {
   require('angular-i18n/angular-locale_de-de');
```

Comment:
Not that it is related to this PR but with all those languages now being part of the core, does this line still make sense or would it need to be generalized?